content/extraProblems.json

```diff
@@ -1164,13 +1164,13 @@
       }
     },
     {
-      "uniqueId": "usaco-1277",
-      "name": "Moo Operations",
-      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1277",
+      "uniqueId": "usaco-1180",
+      "name": "Non-Transitive Dice",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1180",
       "source": "Bronze",
-      "difficulty": "Easy",
+      "difficulty": "Normal",
       "isStarred": false,
-      "tags": ["Greedy"],
+      "tags": ["Complete Search"],
       "solutionMetadata": {
         "kind": "internal"
```

Comment:
why did you completely replace a problem with another

Reply:
i thought i just copy and pasted a problem, and then edited the copy to match with the problem i did.

Reply:
according to the git history, it seems you deleted a problem

Reply:
bump

Reply:
😐 ill fix this pr myself in a few days if they don't respond

Reply:
just do it now